staging/src/k8s.io/code-generator/cmd/import-boss/main.go

```diff
@@ -81,6 +81,7 @@ func main() {
 		"k8s.io/kubernetes/pkg/...",
 		"k8s.io/kubernetes/cmd/...",
 		"k8s.io/kubernetes/plugin/...",
+		"k8s.io/kubernetes/test/e2e/framework/...",
```

Comment:
if we one day move the framework to a separate repository, would it cause an error?

Reply:
That is a nice point.
When separating the framework code into a different repository, I think we will keep the framework code still in k/k repository until finishing to switch to use a new repository from e2e tests.
Then at the time, above line 82 doesn't make any error because we don't need to change k/k code.

After switching to use a new repository code, we will need to remove old framework code in k/k by removing the above line also.

Reply:
ok, SGTM.

Reply:
another note on this.
Just tried this out and technically, I dont think it would cause an issue if the framework was moved out of k/k. though we may still want to update/remove this specific patch if so.